protocols/kad/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # 0.29.0 [unreleased]
 
+- Add `Kademlia::put_record_to` for storing a record at specific nodes,
+  e.g. for write-back caching after a successful read. In that context,
+  peers that were queried in a successful `Kademlia::get_record` operation but
+  did not return a record are now returned in the `GetRecordOk::no_record`
+  list of peer IDs.
```

Comment:
Might be worth mentioning `KademliaCaching` here. What do you think @romanb?

Reply:
:+1: I forgot about the changelog. Updated in https://github.com/libp2p/rust-libp2p/pull/1988/commits/5bada7afbff87335f6bb3991126089ee5502a7e7.